proposals/0088-enable-core-bpf-programs.md

```diff
@@ -62,24 +62,32 @@ published outlining at least the following details:
   program
 - How this changes to this program will be managed after it becomes BPF
 
-**Migrating a native program to core BPF** shall consist of deploying a
-BPF version of the native program to a new arbitrary address and using a
-feature gate to move the BPF program, replacing the existing native program at
-its original program address. No program IDs for existing native programs would
-be changed by this migration process.
+**Migrating a native program to core BPF** shall consist of creating a buffer
+account, owned by the BPF Upgradeable Loader, containing the program's desired
+upgrade authority and the ELF bytes of its BPF implementation. A feature gate
+is used to create the BPF program accounts, replacing the existing native
+program at its original address with a legitimate BPF Upgradeable program.
+
+In the slot immediately following the feature activation, the program will *not*
+be invocable. It will be marked as `DelayedVisibility` in the program cache.
```

Comment:
The program cache and `DelayedVisibility` are internals of the agave client, not protocol. The sentence above is sufficient.

Reply:
Removed.